From: Matthew Garrett <mjg@redhat.com>
Date: Wed Mar 28 14:42:48 2012 -0700
Subject: ipmi: increase KCS timeouts
References: bnc#870618
Patch-mainline: v3.4
Git-commit: 828dc9da50f9632bbc5bc9dfa510619d13135015

IPMI specification suggests to use timeouts of 5 seconds while waiting
for OBF or IBF in KCS. Mainline (3.14-rc7) has 5000000 us, while
SLES11-SP2 still uses 1000000 us, which is quite near to the time some
slow HW needs to prepare an answer.

Signed-off-by: Bodo Stroesser <bstroesser@ts.fujitsu.com>
Acked-by: Jean Delvare <jdelvare@suse.de>

--- a/drivers/char/ipmi/ipmi_kcs_sm.c	2014-03-18 12:30:31.000000000 +0100
+++ b/drivers/char/ipmi/ipmi_kcs_sm.c	2014-03-18 18:46:49.000000000 +0100
@@ -118,8 +118,8 @@ enum kcs_states {
 #define MAX_KCS_WRITE_SIZE IPMI_MAX_MSG_LENGTH
 
 /* Timeouts in microseconds. */
-#define IBF_RETRY_TIMEOUT 1000000
-#define OBF_RETRY_TIMEOUT 1000000
+#define IBF_RETRY_TIMEOUT 5000000
+#define OBF_RETRY_TIMEOUT 5000000
 #define MAX_ERROR_RETRIES 10
 #define ERROR0_OBF_WAIT_JIFFIES (2*HZ)
 
